### PR TITLE
Next to Master

### PR DESCRIPTION
**Pull request by abhinav-from-contentstack**
- updated version to 1.3.1
- Text level rendering support
- Dependencies update 
- Multiple array support

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.3.1](https://github.com/contentstack/contentstack-utils-javascript/tree/v1.3.1) (2023-12-13)
```
Comment (ishaileshmishra) [inline CHANGELOG.md:3]: In the changelog the version is listed as 1.3.1 following 1.1.0. are we missing any released in between ? @harshithad0703

**Review by ishaileshmishra — CHANGES_REQUESTED** [review]
added comment @harshithad0703

File: CHANGELOG.md (hunk 1 above)
Comment (harshithad0703) [reply CHANGELOG.md:3]: ChangeLog was not updated since past releases, will take this up and update in next release

**Review by ishaileshmishra — APPROVED** [review]
approved @harshithad0703 


thank you

**Pull request merged**